styles.css

```diff
@@ -802,7 +801,7 @@ form {
   }
 
   /* || WORKS STYLING */
-  
+
   .work-card {
```

Comment:
Great job so far 👍. Please, as requested by the previous reviewer 👨 . Kindly increase the project cards' width and ensure that the images in the cards have a slightly higher height than the content that is aside like in the original design because they appear to be smaller than the original design on your desktop version 😞 .

| Your Design | Figma Design |
| :---:   | :---: | 
| <img width="1470" alt="image" src="https://user-images.githubusercontent.com/73854999/221140653-6f2186e4-0340-418e-ac24-eb36cf75eeca.png">|<img width="1437" alt="image" src="https://user-images.githubusercontent.com/73854999/221140553-0a9be0a3-2d53-43f6-85c8-a9324ad7487a.png">|

I think you can give the main section of the project 100% and let the cards inherit the width... 😊